src/main/java/org/zalando/putittorest/HttpClientFactoryBean.java

```diff
@@ -36,9 +40,17 @@ public void setSocketTimeout(final int socketTimeout) {
         config.setSocketTimeout(socketTimeout);
     }
 
+    @Hack("Hide type to avoid runtime dependency")
```

Comment:
Spring does Magic on the properties of such factory beans including loading the class